modules/geo-layers/src/h3-layers/h3-hexagon-layer.js

```diff
@@ -22,9 +23,27 @@ function getHexagonCentroid(getHexagon, object, objectInfo) {
   return [lng, lat];
 }
 
-function h3ToPolygon(hexId) {
+export function scalePolygon(hexId, vertices, factor) {
+  const [lat, lng] = h3ToGeo(hexId);
+  const vertexCount = vertices.length;
+  log.assert(vertices[0] === vertices[vertexCount - 1]);
```

Comment:
This assert will break the layer of h3-js makes a change in the future. It won't be the user's fault and it won't provide any useful error message. Can we just adapt the for loop based on whether these two points are the same object?

Reply:
Done.